hooks/jenkins/jira-workflow/Jenkinsfile

```diff
@@ -77,52 +69,47 @@ pipeline {
       }
       steps {
         script {
-          // Specify our credentials to use for the steps
-          withCredentials([usernamePassword(credentialsId: '<github_credentials_id>', 
-                              passwordVariable: 'githubToken', 
```

Comment:
this was not needed, as we can directly use `authentication` in the `httpRequest`, and we don't need to wrap it with `withCredentials`